lib/Adapters/RegressionAdapter/hlsStreamRegressionValidator.js

```diff
@@ -0,0 +1,254 @@
+/**
+ * Created by lilach.maliniak on 12/07/2016.
+ */
+var Q = require('q');
+var _ = require('underscore');
+var fs = require('fs');
+var util=require('util');
+var logger = require('../../../common/logger');
+var m3u8Handler = require('../../manifest/promise-m3u8');
+var mkdirp = require('mkdirp');
+var getDirName = require('path').dirname;
+var networkClient = require('./../../NetworkClientFactory').getNetworkClient();
+const crypto = require('crypto');
+const v_utility = require('./validatorUtility');
+const qio = require('q-io/fs');
+const ErrorUtils = require('../../utils/error-utils');
+
+function checksum (str, algorithm, encoding) {
+    return crypto
+        .createHash(algorithm || 'md5')
+        .update(str, 'utf8')
+        .digest(encoding || 'hex')
+}
+
+let logheader1 = function(literals, ...values) {
```

Comment:
this is duplicate code, the logger has already a build in feature of this, please use it